src/main/webapp/script/postcardgen.js

```diff
@@ -0,0 +1,81 @@
+class Postcard {
+  constructor({title = '', message = '', imgUrl = ''}) {
+    this._title = title;
+    this._message = message;
+    this._img = imgUrl;
+  }
+
+  get title() {
+    return this._title;
+  }
+
+  get message() {
+    return this._message;
+  }
+
+  get img() {
+    return this._img;
+  }
+
+  addImageElem() {
+    const imgElem = document.createElement('img');
+    imgElem.className = 'pcard-img';
+    imgElem.setAttribute('src', this._img);
+    
+    return imgElem;
+  }
+
+  addMessageElem() {
+    const msgElem = document.createElement('div');
```

Comment:
Note on naming.

`const message` feels the better name instead of dropping vowels etc.

It's pretty obvious that this is an element anyways.

const ... = ...**createElement**...;

_"Do not use abbreviations that are ambiguous or unfamiliar to readers outside your project, and do not abbreviate by deleting letters within a word."_  https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers